python/setup.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+
+setup(
+    name='ola',
+    version='0.9.5',
```

Comment:
Can we generate this from configure please so the version updates automatically.